tests/testAutopas/CMakeLists.txt

```diff
@@ -9,8 +9,22 @@ add_executable(runTests ${MY_SRC})
 
 target_compile_definitions(runTests PRIVATE)
 
+if(AUTOPAS_ENABLE_COVERAGE)
+  target_compile_options(runTests PRIVATE -O0 --coverage)
```

Comment:
I wouldn't add `-O0` here. This is the responsibility of the `CMAKE_BUILD_TYPE` and we also check that this is set to `DEBUG`

Reply:
done in [889f266](https://github.com/AutoPas/AutoPas/pull/888/commits/889f26618f64e9d66805dd320b2ef8d07f14578a)